docs/core/project-sdk/overview.md

```diff
@@ -0,0 +1,119 @@
+---
+title: .NET Core project SDK overview
+description: Learn about the .NET Core project SDKs.
+ms.date: 02/02/2020
+ms.topic: conceptual
+---
+# .NET Core project SDKs
+
+.NET Core projects are associated with a software development kit (SDK). Each project SDK is a set of MSBuild [targets](/visualstudio/msbuild/msbuild-targets) and associated [tasks](/visualstudio/msbuild/msbuild-tasks) that are responsible for compiling, packing, and publishing code.
+
+## Available SDKs
+
+The following SDKs are available for .NET Core:
```

Comment:
This makes it sound like it's an exclusive list. It should probably mention that you can author your own SDK that can be distributed via NuGet